src/prelude.rs

```diff
@@ -5,4 +5,5 @@ pub use crate::gpio::GpioExt as _stm327xx_hal_gpio_GpioExt;
 pub use crate::hal::digital::v2::{InputPin, OutputPin};
 pub use crate::hal::prelude::*;
 pub use crate::rcc::RccExt as _stm32f7xx_hal_rcc_RccExt;
+pub use crate::rng::RngExt as _stm32f7xx_hal_rng_RngExt;
```

Comment:
This is no longer the best way to do a re-export (it used to be some time ago). Now this is better:
```suggestion
pub use crate::rng::RngExt as _;
```